src/main/java/com/epam/reportportal/auth/integration/github/GitHubClient.java

```diff
@@ -44,7 +46,7 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(GitHubClient.class);
 
-    private static final String GITHUB_BASE_URL = "https://api.github.com";
+    private static final String GITHUB_BASE_URL = ofNullable(System.getenv("GITHUB_API_URL")).orElse("https://api.github.com");
```

Comment:
Let's probably create a env variable in application.yaml 
and inject a concrete value using spring

Reply:
Quick disclaimer, not really a java dev, just took a look at the docs and I think my latest commit is what you meant.  If not that's fine just let me know what needs to be changed :)

Reply:
This looks that it is exactly what i meant